docs/UserGuide.adoc

```diff
@@ -29,174 +29,452 @@ apps.
+command from the 3rd task in Cow
+
 ==== Show tasks assigned to contact: `contacts assigned`
 
-Format: `contacts assigned CONTACT_ID`
+Displays all tasks assigned to the specified contact in the tasks pane.
+Format: `contacts assigned INDEX`
 
-This will display all tasks assigned to the contact in the tasks pane.
+****
+* Displays all tasks assigned to the person at the specified `INDEX`.
+* The index refers to the index number shown in the displayed person list.
+* The index *must be a positive integer* 1, 2, 3, ...
+****
+
+[TIP]
+The person specified at `INDEX` will be highlighted, and the tasks assigned are
+displayed on the displayed tasks list
+
+Examples:
+
+* `contacts assigned 2` +
+Displays all tasks assigned to the 2nd person in the current displayed person list.
+* `contacts find Jane` +
+`contacts assigned 1`
+Displays all tasks assigned to the 1st person in the list of people resulting
+from the `contacts find` command
 
 === Tasks
 
-Format for entering dates is YYYYMMDD. Format for entering times is HHMM
-in 24-hour time. (All commands from AB4 but prepend `tasks`)
+Format for entering dates is `YYYYMMDD`, e.g. `20183112`.
+Format for entering times is `HHMM` in 24-hour time, e.g. `2359`.
+
+[TIP]
+Most tasks commands has similar format to their equivalent in contacts, except
+that they have different prefixes (`tasks` vs `contacts`)
 
 [NOTE]
 You are allowed to have entries with the same name
 
 ==== View all tasks: `tasks list`
 
+Shows a list of all tasks in Cow. +
 Format: `tasks list`
 
 ==== Add task: `tasks add`
 
+Adds a task to Cow. +
 Format:
 `tasks add n/TASK_NAME [sd/START_DATE] [st/START_TIME] ed/END_DATE et/END_TIME [t/TAG]…`
 
+[TIP]
 If start date or time is not entered, the missing field(s) will default to the current date/time.
 
+Examples:
+
+* `tasks add n/Math Assignment sd/20180101 st/0000 ed/20181231 et/2359 t/school t/urgent`
+* `tasks add n/Milk the cows sd/20181001 st/0000 ed/20181129 et/2359 t/farm`
+
 ==== Edit task: `tasks edit`
 
+Edits an existing task in Cow. +
 Format:
-`tasks edit INDEX [n/TASK_NAME] [sd/START_DATE st/START_TIME ed/END_DATE et/END_TIME] [t/TAG]…`
+`tasks edit INDEX [n/TASK_NAME] [sd/START_DATE st/START_TIME] [ed/END_DATE et/END_TIME] [t/TAG]…`
 
+****
+* Edits the task at the specified `INDEX`. The index refers to the index number shown in the displayed task list. The index *must be a positive integer* 1, 2, 3, ...
+* At least one of the optional fields must be provided.
+* Existing values will be updated to the input values.
+* When editing tags, the existing tags of the person will be removed i.e adding of tags is not cumulative.
+* You can specify `t/` to indicate that you want to clear all existing tags.
+* Date and time are regarded as one field, and after editing, start DateTime
+must be before end DateTime.
+****
+
+Examples:
+
+* `tasks edit 1 sd/20050108 st/1235` +
+Edits the start DateTime of the 1st task to be 8 January 2005, 12.35pm.
```

Comment:
Should `DateTime` be `date and time` instead? Also, "12:35pm" with a colon is more standard